core/src/main/java/org/hisp/dhis/android/core/event/internal/EventDateUtils.kt

```diff
@@ -27,17 +27,20 @@
  */
 package org.hisp.dhis.android.core.event.internal
 
+import kotlinx.datetime.*
+import kotlinx.datetime.TimeZone
+import org.hisp.dhis.android.core.arch.helpers.DateUtils.atStartOfDayInSystem
 import org.hisp.dhis.android.core.event.Event
 import org.hisp.dhis.android.core.event.EventStatus
 import org.hisp.dhis.android.core.period.PeriodType
+import org.hisp.dhis.android.core.period.clock.internal.ClockProvider
 import org.hisp.dhis.android.core.period.internal.PeriodHelper
 import org.koin.core.annotation.Singleton
-import java.util.*
-import java.util.concurrent.TimeUnit
 
 @Singleton
-class EventDateUtils(
+internal class EventDateUtils(
```

Comment:
Did you check this class wasn't used by the Android app?

Reply:
Yes, there are no usages of this class.